src/ngx_http_lua_util.c

```diff
@@ -4324,7 +4324,7 @@ ngx_http_lua_log_escape(u_char *dst, u_char *src, size_t size)
         size--;
     }
 
-    return NGX_OK;
+    return 0;
```

Comment:
imho, this is a more correct return value given the `size_t` return type of this function.